### DISCUSSION TITLE
tutorials/intro-typescript/07-listings-rest-api

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#164] [course feedback]
# tutorials/intro-typescript/07-listings-rest-api

https://www.apollographql.com/tutorials/intro-typescript/07-listings-rest-api

**Comment by andreashouben** [comment]
If you add the `override` keyword to the baseURL you'll get a compiler error if you misspell the variable like I did (I wrote `baseUrl`).

**Reply by cunisia** [reply]
Saved me a lot of trouble shooting, thanks !

**Reply by lizhennessy** [reply]
For reference for other learners:
```
override baseURL = "https://rt-airlock-services-listing.herokuapp.com/";
```

**Reply by brunosm16** [reply]
Great content, i'm learning GraphQL very fast.
